src/CovidCountyDash.jl

```diff
@@ -89,6 +89,8 @@ function download_and_preprocess()
                     29998
                 elseif state == "Missouri" && county == "Joplin"
                     29997
+		elseif state == "American Samoa"
+		    03999
```

Comment:
I think this should be 60999?  Do we also need its population?

Reply:
Oh, 60999 should be right - I looked up the wrong row in the FIPS table.  And yes, we should fill in the population; I can update this PR for that when I have time.

Reply:
Aha, yeah, and it looks like we don't need to special-case its FIPS here — once it has a population entry the unknown FIPS will get assigned appropriately by the next branch here.

Reply:
Yeah, my fault for not looking a little deeper.  Please recheck the samoa branch, this should be right now.